packages/core/bootstrap/src/lib/ws/reducer.ts

```diff
@@ -14,11 +14,14 @@ export interface ConnectionsState {
   connecting: {
     [key: string]: number // Filter if its more than one
   }
+  wasEverConnected: {
```

Comment:
Connections state is growing a lot, I think it'd be better to have the same structure as subscriptions, so we have all the connection info in one place
```js
state = {
[key: string]: {
    active: boolean
    wasEverConnected?: boolean
    connecting: number
  }
}
```
What do you guys think?

Reply:
I agree, let's structure them similarly to subscriptions state, but let's do it in the new PR.